code/modules/mob/living/carbon/human/species_types/flypeople.dm

```diff
@@ -73,7 +73,7 @@
 	if(locate(/datum/reagent/consumable) in reagents.reagent_list)
 		var/mob/living/carbon/body = owner
 		// we do not loss any nutrition as a fly when vomiting out food
-		body.vomit(0, FALSE, FALSE, 2, TRUE, force=TRUE, purge=TRUE)
+		body.vomit(0, FALSE, FALSE, 2, TRUE, force=TRUE, purge_ratio = 0.67)
```

Comment:
Undocced change?

Reply:
no mechanical change, previously purge=TRUE was equivalent to setting the ratio to 0.67